default.json

```diff
@@ -103,6 +103,15 @@
       "datasourceTemplate": "docker",
       "versioningTemplate": "docker"
     },
+    {
+      "description": "Update Github Action references in the README.md on renovatebot/github-action repository",
+      "fileMatch": ["README.md"],
+      "matchStrings": [
+        "insert regex that matches actions/checkout@v2.3.4 and matches renovatebot/github-action@v25.51.5"
```

Comment:
Will need a proper regex that matches the 2 references we have. Maybe we should split this into 2 parts, one for `actions/checkout` and one for `renovatebot/github-action`?

Reply:
```suggestion
        "(?<depName>actions\\/checkout)@(?<currentValue>v\\d+\\.\\d+\\.\\d+)",
        "(?<depName>renovatebot\\/github-action)@(?<currentValue>v\\d+\\.\\d+\\.\\d+)"
```